README.rst

```diff
@@ -23,7 +31,8 @@ Requirements
 ------------
 
 A C++ compiler that supports C++14. Clang version 5 is a good pick, GCC version
```

Comment:
Maybe a section about frozen testing (How to build/run tests, C+17 / GoogleBenchmark need, ...) could be added to the README ?

Reply:
Correct! will do.